MapboxCoreNavigation/MBRouteController.h

```diff
@@ -82,3 +83,9 @@ extern const MBRouteControllerNotificationUserInfoKey MBRouteControllerIsProacti
 @interface NSString (MD5)
 - (NSString * _Nonnull)md5;
 @end
+
+@interface MBNavigator (additions)
+
+- (NSUInteger)setupRouter:(NSString *)tilesPath translationsPath:(NSString *)translationsPath;
```

Comment:
Documentation would be great here. It wasn’t obvious to me that the return value is a tile count. From the implementation of this method, I initially got the impression that it was an error code of some sort.

Reply:
This comment is not relevant any longer. `configureRouter` is not wrapped in `setUpRouter` but instead called directly from an initializer in Swift which is documented.